guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -64,16 +65,23 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
      */
     var MENU_DRAG_VERTICAL_TOLERANCE = 10;
 
-    /*
+    /**
      * In order to open the guacamole menu, we need to hit ctrl-alt-shift. There are
      * several possible keysysms for each key.
      */
     var SHIFT_KEYS  = {0xFFE1 : true, 0xFFE2 : true},
         ALT_KEYS    = {0xFFE9 : true, 0xFFEA : true, 0xFE03 : true,
                        0xFFE7 : true, 0xFFE8 : true},
         CTRL_KEYS   = {0xFFE3 : true, 0xFFE4 : true},
+        END_KEYS    = {0xFF57 : true, 0xFFB1 : true},
         MENU_KEYS   = angular.extend({}, SHIFT_KEYS, ALT_KEYS, CTRL_KEYS);
 
+    /**
+     * Keysym for sending the DELETE key when the Ctrl-Alt-End hotkey
+     * combo is pressed.
+     */
+    var DEL_KEY     = 0xFFFF;
```

Comment:
Need an `@type` tag. I also suggest just separating `DEL_KEY` and `=` with a single space, seeing as it's not being aligned with anything any more.

Reply:
Fixed.